packages/meditrak-server/src/routes/importSurveyResponses/importSurveyResponses.js

```diff
@@ -90,6 +92,7 @@ export async function importSurveyResponses(req, res) {
                   start_time: surveyDate,
                   end_time: surveyDate,
```

Comment:
Can we be cheeky and slip in Juliana's suggestion here? Set the start and end times for new survey responses to the current time in the user's timezone?